CHANGELOG.md

```diff
@@ -1,67 +1,78 @@
-x.x.x Release notes (yyyy-MM-dd)
```

Comment:
I suggest ignoring this file in your markdown linter 🙂

Reply:
Oh no... 🙈

Reply:
My :+1: is based on Kræn's suggestion above 😄

Reply:
Done